tests/foreman/cli/test_role.py

```diff
@@ -134,6 +134,74 @@ def test_positive_update_name(self):
                 role = Role.info({'id': role['id']})
                 self.assertEqual(role['name'], new_name)
 
+    @tier1
+    def test_positive_list_filters_by_id(self):
```

Comment:
What is a big difference between this and `test_positive_create_with_permission`?